client/my-sites/site-settings/form-writing.jsx

```diff
@@ -52,6 +52,10 @@ class SiteSettingsFormWriting extends Component {
 		);
 	}
 
+	isMobile() {
```

Comment:
Looks pretty solid :)

I've got some tinsy tiny nit picks here though...
- Semantics: iP(ad) isn't mobile as such - perhaps we could rename to `isTouchDevice`? That might better describe the intent for this check actually since it's the `touch`ing that causes the issue.
- Reuse: Feels like this would be useful elsewhere

Neither are blockers though since a quick scan of Calypso shows that this is only used once elsewhere (`analytics/ad-tracking.js`, with slight deviation) and the semantics still makes plenty of sense 🤷‍♂️

Reply:
I am not very sold on the `isTouchDevice` since Windows/Linux laptops/tablets already support touch screens and touch events, but at the same time they support D'n'D on the bookmarks bar. 

At the same time the iPad is not as mobile, but doesn't support D'n'D on the bookmarks bar. 

I agree the naming is not the best. There's already a `.touch` class at the root elements, which is added for touch-supported devices, but for the first reason above I refrained from using it :(

It would be awesome to extract it in a library, but we can do that later, when we decide on a proper place to add it. I copied it over from `ad-tracking.js` since it's just a line of code and didn't think of a good place to extract it. Also it's not exactly the same as the one used there :)